src/CoreBluetooth/CBPeer.cs

```diff
@@ -10,8 +10,20 @@
 namespace CoreBluetooth {
 	public partial class CBPeer  {
 #if !NET
+#if NET
+		[SupportedOSPlatform ("ios8.0")]
+		[SupportedOSPlatform ("macos10.13")]
+		[UnsupportedOSPlatform ("ios7.0")]
```

Comment:
We should remove the supported one, correct?

Reply:
Nope on both (ironically enough TJ raised this point yesterday). The plan we made is to convert all attributes, no matter how old, to keep the conversion straight forward and 1:1, and then write a tool long term to strip the "too old" attributes, since we'll need that as we move to NET's "every year drop one version" policy.

Reply:
I agree, but does [UnsupportedOSPlatform ("ios7.0")] and then [SupportedOSPlatform ("ios8.0")] make sense? How does the analyser behave in this case?